wasm/all_test.go

```diff
@@ -0,0 +1,94 @@
+package wasm
```

Comment:
@Rizary name if the test file is incorrect. Tests should be attached to a specific file

Reply:
Should it? I mean we can have one test that might be imported in another test files, right? Because it's kinda redundant if I write setup twice. Or it can be confusing if we put setup in one of the test file. What do you suggest?

Reply:
_test.go should belong to something, not just an abstract test. You can move these tests to one of the proxy_test or sdk_test if they not belong to any actuall file